packages/employer-api/src/services/claim.service.ts

```diff
@@ -88,6 +88,32 @@ export const insertClaim = async (
     return false
 }
 
+export const insertLegacyClaim = async (
+    id: string,
+    userGuid: string,
+    submissionID: string,
+    catchment: number,
+    storefront: number,
+    trx?: any
+) => {
+    const data = {
+        id,
+        form_submission_id: submissionID,
+        created_date: new Date(),
+        created_by: userGuid,
+        status: "Draft",
+        catchmentno: catchment,
+        workbc_centre: storefront
```

Comment:
workbc_centre needs to be set to catchmentNo-storefrontId, not storefrontId. This is causing the workbc column in the UI to always display 'Unassigned' for these claim forms.